src/index.js

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from 'react-router-dom';
 import './index.css';
-import App from './App';
+import CalculatorPage from './components/CalculatorPage';
+import Quote from './components/Quote';
+import Home from './components/Home';
+import Header from './components/Header';
+
 import reportWebVitals from './reportWebVitals';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root'),
+const rootElement = document.getElementById('root');
+render(
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<Header />}>
+        <Route path="/" element={<Home />} />
+        <Route path="calculator" element={<CalculatorPage />} />
```

Comment:
[OPTIONAL] I think your calculator component has some issues that you might want to look into it. The first one is that the size of the button does not match the actual size of the button, which causes the user to need to click on the exact spot where the button exists. This might be very frustrating as some of the operations don't show up on the screen either.

Recommendations:
1. You could try to make the button occupy the entire space of the actual box.
2. Display the operations on the screen of the calculator as well.

Screenshot for reference of the actual size of the button:
![image](https://user-images.githubusercontent.com/79114233/149170471-7b0d9bb6-0f6d-4111-ad64-3fabe7c7ef4b.png)

[OPTIONAL] I think the `±` operator does not work at all. I consider it important, please fix this as you will have to write tests for the calculator on a future milestone and all of the operations should be tested as well.

Reply:
Thanks, I have made the changes you discussed :)